app/frontend/src/features/auth/useAuthStore.ts

```diff
@@ -73,12 +73,8 @@ export default function useAuthStore() {
         setLoading(true);
         try {
           const auth = await service.user.passwordLogin(username, password);
+          setUserId(auth.userId);
 
-          if (!auth.jailed) {
-            const user = await service.user.getUser(username);
-            setUserId(user.userId);
-            queryClient.setQueryData(["user", user.userId], user);
```

Comment:
I just looked at how this differs from `getCurrentUser` - `getCurrentUser` calls the `Ping` rpc to get the current user, so I think we could do the same here and we wouldn't even need to change the API?